Tasks/PublishBuildArtifacts/publishbuildartifacts.ts

```diff
@@ -68,7 +68,7 @@ async function run() {
 
         // upload or copy
         if (artifactType === "container") {
```

Comment:
I think this is a good opportunity to rename the task value for "Visual Studio Team Services/TFS" to `service` or possibly `drop`. I know we don't have to, but continuing to have `container` in the build definition will be confusing.

As far as people switching from v1 to v2: We're allowed to make breaking changes in new major versions, but we could check `if (artifactType === "filepath") {` first and handle other values (`service` or `container`) in the else clause.